src/api/moviesApi.js

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+
+const API_URL = 'http://localhost:4000';
```

Comment:
Do you have an actual API? This will be a part of one of the future chapters. If you don't have API now you can return `Promise.Resolve`, it will be async and there won't be any difference in how you treat it in code.

Reply:
Yes, i'm using the MoviesAPI.ReactJS project that comes with the course.  It starts on localhost:4000